src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -312,8 +312,9 @@ THREE.WebGLShadowMap = function ( _renderer, _lights, _objects ) {
 
 		if ( ! customMaterial ) {
 
-			var useMorphing = geometry.morphTargets !== undefined &&
-					geometry.morphTargets.length > 0 && material.morphTargets;
+			var useMorphing = ( ( geometry.morphTargets !== undefined && geometry.morphTargets.length > 0 ) || 
+				( geometry.morphAttributes.position !== undefined && geometry.morphAttributes.position.length > 0 ) ) 
+				&& material.morphTargets;
```

Comment:
>geometry.morphAttributes.position !== undefined

This will throw an error if `geometry.morphAttributes` is not defined. You can test this on the three.js examples.

An alternate pattern is `if ( geometry instanceof THREE.BufferGeometry )`. I am not sure which pattern is preferred. /ping @mrdoob